packages/express-backend/backend.js

```diff
@@ -17,7 +17,7 @@ const supabaseKey = process.env.SUPABASE_KEY;
 const supabase = createClient(supabaseUrl, supabaseKey);
 
 const corsOptions = {
-  origin: process.env.BACKEND_URL,
+  origin: "*",
```

Comment:
It is still showing this change despite the rollback commit. Can you double check that it won't be merged in if it doesn't work.